Services/Object/classes/class.ilObject.php

```diff
@@ -2320,4 +2331,14 @@ public function getPossibleSubObjects($a_filter = true)
     {
         return $this->objDefinition->getSubObjects($this->type, $a_filter);
     }
+
+    /**
+     * @param $a_target_id
+     * @param $a_copy_id
+     * @return void
+     */
+    public function handleChildCloning($a_target_id, $a_copy_id)
```

Comment:
```suggestion
    public function handleChildCloning($a_parent_ref_id, $a_child_ref_id)
```

I think this would clarify what the parameters mean here...

Reply:
Good point, I changed the parameter names for clarification, thx!